src/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,65 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"github.com/vshn/provider-exoscale/apis"
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	kubernetes "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
having pkg `kubernetes` and aliasing a client to `kubernetes` as well creates confusion
```suggestion
	client "sigs.k8s.io/controller-runtime/pkg/client"
```

Reply:
yes that's an error here, I removed completely the alias as it's already `client`